nerfstudio/process_data/metashape_utils.py

```diff
@@ -188,10 +191,21 @@ def metashape_to_json(
 
     data["frames"] = frames
 
+    summary = []
+
+    if isinstance(ply_filename, Path):
```

Comment:
I believe a proper implementation (similar to what we did for colmap) is to save a "applied_transform" in the data dict (e.g. see [this line](https://github.com/nerfstudio-project/nerfstudio/blob/main/nerfstudio/process_data/colmap_utils.py#L479) in colmap utils), and then apply the applied_transform to point cloud (see [this line](https://github.com/nerfstudio-project/nerfstudio/blob/main/nerfstudio/process_data/colmap_utils.py#L690-L692) in colmap case).

Do you have a sample metashape exported data that I can play? I believe I can help make it in the way I suggested.

Reply:
Sure! I am not at home at the moment, but I can provide data in a few hours

Reply:
Here is a very small example: https://drive.google.com/file/d/1GQBYRRQO7opLUqQ2u7z-udcxrfC7zHaj/view?usp=sharing

Reply:
Damn there is at least one misaligned camera... maybe it still can serve this purpose here. Else I could revisit and fix it. I am not good at using metashape yet. Still in trial phase and thinking if I should buy it or not.

Reply:
Can you try my revision here (I built on top of your changes) and verify it works for you? https://github.com/jb-ye/nerfstudio/tree/metashape_splatfacto